app/jobs/import_url_with_file_use_job.rb

```diff
@@ -0,0 +1,34 @@
+require 'uri'
+require 'tmpdir'
+require 'browse_everything/retriever'
+
+# Given a FileSet that has an import_url property,
+# download that file and put it into Fedora
+# Called by AttachFilesToWorkJob (when files are uploaded to s3)
+# and CreateWithRemoteFilesActor when files are located in some other service.
+class ImportUrlWithFileUseJob < ::ImportUrlJob
```

Comment:
What change was needed here that required us overriding the (presumably) upstream `ImportUrlWithFileUseJob`?